frontend/App.tsx

```diff
@@ -1,78 +1,134 @@
-import { StyleSheet, Text, View, Button } from 'react-native';
-// import { Button } from "react-bootstrap";
+import React, { Component } from "react";
+import { Pressable, StyleSheet, Text, View, Button } from 'react-native';
+
 import { NavigationContainer } from '@react-navigation/native';
 import { createNativeStackNavigator } from '@react-navigation/native-stack';
+// import MaterialIcons from "@expo/vector-icons/MaterialIcons";
+
 import { SearchContacts } from './components/SearchContacts';
 import { ChatPanel } from './components/ChatPanel';
+import { Regstyles, Registration } from './components/Registration';
+import { readData, removeData } from './services/AsyncStorage';
+// import { PhoneinputTest } from './components/PhoneinputTest'; 
 
- 
-export default function App() {
-  // const url = "https://gner4se1je.execute-api.ap-south-1.amazonaws.com/Prod/func1"
-  const func1_url = "http://127.0.0.1:3000/func1";
-  const func2_url = "http://127.0.0.1:3000/func2";
-  
-  let [func1_resp, setFunc1Resp] = useState({});
-  let [func2_resp, setFunc2Resp] = useState({});
+const func1_url = "http://127.0.0.1:3000/func1";
+const func2_url = "http://127.0.0.1:3000/func2";
 
-  const [current, setCurrent] = useState('HomeScreen');
+const STORAGE_KEY = '@save_phone'
 
-  // useEffect(() => {
-  //     axios.get(url)
-  //     .then(response => response.data)
-  //     .then(json => {
-  //       console.log(json);
-  //     setResp(json);
-  //   })
-  // })
-  
-  // passing an empty array as second argument triggers the callback in 
-  // useEffect only after the initial render thus replicating `componentDidMount` lifecycle behaviour
+const Stack = createNativeStackNavigator();
 
-  useEffect(() => {
-    axios.get(func1_url)
-    .then(response => setFunc1Resp(response.data))
-  },[]);
+type State = {
+  phonenumber: string | undefined,
+  func1_resp: {},
+  func2_resp: {},
+  current: string,
+};
 
-  // useEffect(() => {
-  //   axios.get(func2_url)
-  //   .then(response => setFunc1Resp(response.data))
-  // },[]);
+export default class App extends Component<{}, State> {
+  constructor(props: {}) {
+    super(props);
+    this.state = {
+      phonenumber: "",
+      func1_resp: {},
+      func2_resp: {},
+      current: "",
+    }
+     // const url = "https://gner4se1je.execute-api.ap-south-1.amazonaws.com/Prod/func1"
+  }
 
-  function HomeScreen({ navigation }) {
-    return (
-      <View style={{ flex: 1, alignItems: 'center', justifyContent: 'center' }}>
-        <Text>Home Screen</Text>
-        <Button
-          title="SearchContacts"
-          onPress={() => navigation.navigate('SearchContacts')}
-        />
-        <Button
-          title="Chat with me"
-          onPress={() => navigation.navigate('ChatPanel')} />
-      </View>
-    );
+  componentDidMount(){
+    console.log("inside mount")
+    readData(STORAGE_KEY)
+      .then( (mphone) => {this.setState({phonenumber: mphone})})
```

Comment:
use camel casing for variable names - phoneNumber

Reply:
ok